server/opendp_apps/dataset/views.py

```diff
@@ -18,7 +19,18 @@ def get_queryset(self):
         return self.queryset.filter(creator=self.request.user)
 
 
-class DepositorSetup(BaseModelViewSet):
+class DepositorSetupViewSet(BaseModelViewSet):
```

Comment:
- List endpint:
  - Doesn't have any auth associated with it
    - e.g. http://127.0.0.1:8000/api/deposit/ - shows all objects even if not logged in
  - Check that the user making the request is the DataSet's creator or superuser

- Update endpoint:
  - Check that the user making the request is the DataSet's creator (superuser doesn't have privileges)

Reply:
Auth permissions are handled in #199, and new permissions class checks that the user is the creator when attempting to modify an individual DataSet